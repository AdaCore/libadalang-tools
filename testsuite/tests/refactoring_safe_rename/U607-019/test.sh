safe_rename -P default -S baz.ads -L 3 -R 18 -N B --algorithm analyse_ast
