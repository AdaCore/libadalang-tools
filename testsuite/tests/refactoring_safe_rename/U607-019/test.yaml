description: |
    This test checks if the safe rename tool correctly handles projects
    with files that only contain the No_Body pragma.

driver: shell_script
