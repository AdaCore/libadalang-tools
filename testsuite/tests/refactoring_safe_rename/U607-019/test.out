# Renaming node baz.ads:3:18: 
References:
Unit:baz.adb
Line: 3
Source Location Range:
Start_Line: 3
Start_Column: 18
End_Line: 3
End_Column: 19
Line: 5
Source Location Range:
Start_Line: 5
Start_Column: 19
End_Line: 5
End_Column: 20
Unit:baz.ads
Line: 3
Source Location Range:
Start_Line: 3
Start_Column: 18
End_Line: 3
End_Column: 19

